src/Symfony/Component/Routing/Generator/Dumper/CompiledUrlGeneratorDumper.php

```diff
@@ -41,6 +41,25 @@ public function getCompiledRoutes(): array
         return $compiledRoutes;
     }
 
+    public function getCompiledAliases(): array
+    {
+        $routes = $this->getRoutes();
+        $compiledAliases = [];
+        foreach ($this->getRoutes()->getAliases() as $name => $alias) {
+            $deprecation = $alias->isDeprecated() ? $alias->getDeprecation() : [];
+            while ($routes->hasAlias($alias->getTarget())) {
+                $alias = $routes->getAlias($alias->getTarget());
+            }
```

Comment:
What is an alias targets a deprecated alias. Would I get a deprecation? Should I?

Reply:
You won't get it with the current version.

Reply:
Right, so should I get one?

Reply:
I don't have a strong opinion on this one, but I would lean to a yes.

Reply:
Me too.

Reply:
Done.